### DISCUSSION TITLE
Managing new release updates of the COE starter toolkit

### DISCUSSION THREAD

**Discussion by gcusworth** [microsoft/powerapps-tools#991] [Q&A] [answered]
Is there a recommended method for installing updated versions of the COE starter toolkit whilst keeping edits/customisations on a previous version? Or is the only option to replicate the changes on the newly installed updated version? Thanks

**Comment by manuelap-msft** [comment, marked as answer]
Please see https://docs.microsoft.com/en-us/power-platform/guidance/coe/setup#installing-updates and https://docs.microsoft.com/en-us/power-platform/guidance/coe/setup#extending-the-starter-kit

**Reply by jw97305** [reply]
Unfortunately this doesn't really answer the question, these instructions are very vague and require expert knowledge of ALM. Are there any step-by-step instructions for specifically extending the starter kit?

**Reply by manuelap-msft** [reply]
Yes, extending the kit would require detailed knowledge of solutions, solution layers and ALM. Here is a good starting point https://docs.microsoft.com/en-us/power-platform/alm/

It's a complex topic, and not in our charter to provide more detailed instructions, many customers work with partners to extend the kit.
